### PR TITLE
Edit readme and build guide in markdown (monero-GUI-guide.md)

### PR DESCRIPTION
**Pull request by erciccione**
Build the markdown version of the guide, getting it ready for 1.6, so that users can already use it (only #132 need to be added at the moment)

### CODE REVIEW

**Comment by erciccione** [comment]
@el00ruobuob @SamsungGalaxyPlayer @dEBRUYNE-1 You can see from the built markdown how the new guide will look like, feel free to suggest changes

**Comment by el00ruobuob** [comment]
I can see a few glitches on some lists (missing double space at the end maybe) but it looks good overall

**Comment by erciccione** [comment]
@el00ruobuob Could you point to those glitches? i must have missed them.

File: monero-GUI-guide.md [hunk 1]
```diff
@@ -66,6 +82,23 @@ Windows Preparation
...
+
+A miner is included in the Monero GUI software and, as a result, most
+anti virus software flags the Monero GUI files as malware and
+subsequently quarantines them. Fortunately, however, there's a fairly
+trivial work around you can use. That is:\
++ Create a new directory / folder.\
++ Open your AV software.\
++ Add an exception for the newly created directory / folder. Put
+differently, you have to whitelist the newly created directory /
+folder.\
```
Comment (el00ruobuob) [inline monero-GUI-guide.md]: idk where those `\` are comming from: maybe trailing 4*spaces?

File: monero-GUI-guide.md [hunk 2]
```diff
@@ -478,56 +760,98 @@ The `Settings` tab provides tools for customizing configuration options.
...
+
+(1) **Hide balance:** If you are running your wallet in public, you may
+    want to hide its balance. This can be useful for point of sales
+    systems.\
+(2) **Enable Payment ID:** Long payment IDs are disabled by default.
+    Click here to enable it. This can be useful for transfers to
+    exchanges that require a long payment ID such as Binance (at the
+    time of writing). Using long payment IDs is detrimental to
+    your privacy. Please ask services that still use this deprecated
+    feature to use subaddresses instead.
```
Comment (el00ruobuob) [inline monero-GUI-guide.md]: Missing trailing 2*space here (my bad)

File: monero-GUI-guide.md [hunk 3]
```diff
@@ -316,24 +513,76 @@ Receive Monero
...
+(5) **Transaction list:** Here are displayed your transactions on
+    this account. Each transaction is displayed with it's informations:
+    `Amount`, `Tx ID`, `Height`, `Date`, `Fee` (for outgoing
+    transactions), and `Description` *Optional*.\
+(6) **Set description:** You can optionally click here to set a
+    description of your choice to this transaction.\
+(7) **Prove transaction:** Clicking here will generate a payment proof
+    in case of a dispute. More details on proving payments can be found
+    [here](https://getmonero.org/resources/user-guides/prove-payment.html)
+    *For outgoing transaction only*
```
Comment (el00ruobuob) [inline monero-GUI-guide.md]: Missing trailing 2*spaces

File: monero-GUI-guide.md [hunk 4]
```diff
@@ -90,14 +123,83 @@ that page  
...
+Welcome
+=======
+
+![Welcome](media/wizard_0-welcome.png)
+
+Extract the package and click on `monero-wallet-gui`.
+
+Welcome to Monero GUI Wallet.
+
+(1) **Language:** Click here to see a list of available languages.
```
Comment (el00ruobuob) [inline monero-GUI-guide.md]: Missing trailing 2*spaces

**Comment by el00ruobuob** [comment]
A proper review of the *issue*

File: monero-GUI-guide.md (hunk 1 above)
Comment (erciccione) [reply monero-GUI-guide.md]: It's caused by the 2*spaces. They are not needed in a list

**Comment by erciccione** [comment]
Thanks for the review @el00ruobuob . I added the changes to the related chapters and rebuilt the guide.

**Pull request merged**